src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -1522,6 +1522,11 @@ pub const NGROUPS: usize = 16;
 pub const RB_PAUSE: ::c_int = 0x40000;
 pub const RB_VIDEO: ::c_int = 0x20000000;
 
+// For getrandom()
+pub const GRND_NONBLOCK: ::c_uint = 0x0001;
+pub const GRND_RANDOM: ::c_uint = 0x0002;
+pub const GRND_INSECURE: ::c_uint = 0x0004;
+
```

Comment:
We might want to also move the constant declarations to `src/unix/bsd/freebsdlike/mod.rs` so that all the `getrandom` stuff is all in one place.